.eslintrc.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: 'airbnb-base',
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  rules: {
+    "no-underscore-dangle": "warn",
+    "vars-on-top": "warn",
+    "no-var": "warn",
```

Comment:
don't mute it

Reply:
according to these rules - https://github.com/eslint/eslint/blob/master/lib/rules/no-var.js  "no-var" worked correctly and everywhere show the errors that we agreed to edit in new branches